.env.sample

```diff
@@ -23,6 +23,8 @@ PELIAS_ADDRESS_PARSER_URL=http://localhost:6800/parser/parse
 NN_ASSIGN_PSK=localdev
 QUERY_PSK=localdev
 
+SITE_BASE_URL=https://db.mesh.nycmesh.net
```

Comment:
@james-otten for non-secret env variables is there a different procedure than the one from #455 ?

Reply:
https://github.com/nycmeshnet/meshdb/pull/476